transactions/demo-rust/.config/nextest.toml

```diff
@@ -0,0 +1,5 @@
+[profile.default]
```

Comment:
As we're using ogmios and plutip which are using some set ports/folders, parallel testing fails.
I have 3 places to configure this, so things work in every environment:
- `serial_test` crate and `#[serial]` macro for `cargo test`
- this configuration for `nextest` locally, without using nix
- `cargoNextestExtraArgs` in the `build.nix` file for nix (the above configuration is not included, `cleanCargoSource` filters the `.config` folder)

Reply:
I have no idea what this is, and will probably not know what this is next time I come here. If it's needed then ok, otherwise remove everything that's not necessary